src/iiiftilesource.js

```diff
@@ -393,6 +393,9 @@ $.extend( $.IIIFTileSource.prototype, $.TileSource.prototype, /** @lends OpenSea
         iiifTileSizeHeight = Math.ceil( tileHeight / scale );
         if (this.version === 1) {
             iiifQuality = "native." + this.tileFormat;
+        }
+        else if (this.tileQuality) {
```

Comment:
Please put the `else if` on the same line as the closing curly bracket, to match our code style.